packages/atomic/src/components/facets/facet-segmented-value/facet-segmented-value.tsx

```diff
@@ -0,0 +1,46 @@
+import {FunctionalComponent, h} from '@stencil/core';
+import {Button} from '../../common/button';
+import {FacetValueProps} from '../facet-common';
+
+export const FacetSegmentedValue: FunctionalComponent<FacetValueProps> = (
+  props,
+  children
+) => {
+  const compactCount = new Intl.NumberFormat(props.i18n.language, {
+    notation: 'compact',
+  }).format(props.numberOfResults);
+
+  const count = props.numberOfResults.toLocaleString(props.i18n.language);
+
+  const ariaLabel = props.i18n.t('facet-value', {
+    value: props.displayValue,
+    count: props.numberOfResults,
+  });
+
+  return (
+    <li key={props.displayValue}>
+      <Button
+        style="outline-bg-neutral"
```

Comment:
You should create a new style for the button that fits the mockups.

see `src/components/common/button-style.tsx` & `src/global/global.pcss`

That should fix the issue you have with the overrides

Reply:
Sounds good